sdk/metric/provider.go

```diff
@@ -68,6 +69,10 @@ func NewMeterProvider(options ...Option) *MeterProvider {
 //
 // This method is safe to call concurrently.
 func (mp *MeterProvider) Meter(name string, options ...metric.MeterOption) metric.Meter {
+	if name == "" {
+		global.Warn("Meter name should not be empty. Empty meter name is specified as invalid.")
```

Comment:
This seems like a pretty verbose way to say the same thing as the following.

```suggestion
		global.Warn("Invalid Meter name: \"\"")
```